frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -57,13 +59,20 @@ const ReviewPage = () => {
   const [commentLine, setCommentLine] = useState(-1);
 
   const [returnToTranslator, setReturnToTranslator] = useState(false);
+  const [submitTranslation, setSubmitTranslation] = useState(false);
 
-  const closeModal = async () => {
+  const closeReturnToTranslatorModal = async () => {
```

Comment:
nit: i think the async is unnecessary here, and in all the other related functions